docs/source/docs/quick-start/wiring.md

```diff
@@ -0,0 +1,28 @@
+# Wiring
+
+## Coprocessor with regulator
+
+1. **IT IS STRONGLY RECOMMENDED** to use one of the recommended power regulators to prevent vision from cutting out from voltage drops while operating the robot. We recommend wiring the regulator directly to the power header pins or using a locking USB C cable. If neither of these work for you at least heavily hot glue the connector.
+
+2. Run an ethernet cable from your Pi to your network switch / radio.
+
+This diagram shows how to use the recommended regulator to power a coprocessor.
+
+```{image} images/pololu-diagram.png
```

Comment:
This just shows magical V+ and GND pins on the coprocessor. should we specify what these are exactly for supported hardware? EG a usb cable, 0.1in pitch connectors, etc.